tests/conf/deepglobelandcover_0.yaml

```
@@ -0,0 +1,19 @@
+experiment:
+  task: "deepglobelandcover"
+  module:
+    loss: "ce"
+    segmentation_model: "unet"
+    encoder_name: "resnet18"
+    encoder_weights: null
+    learning_rate: 1e-3
+    learning_rate_schedule_patience: 6
+    verbose: false
+    in_channels: 3
+    num_classes: 7
+    num_filters: 1
+    ignore_zeros: False
```

```suggestion
    ignore_index: null
```
Should we also have a `conf/deepglobelandcover.yaml` file? I guess that could be added later after we find a good set of hyperparams.